azurelinuxagent/distro/default/rdma.py

```diff
@@ -0,0 +1,74 @@
+# Windows Azure Linux Agent
+#
+# Copyright 2016 Microsoft Corporation, SUSE LLC, Robert Schweikert
```

Comment:
It appears like our guidance on this is: new source files should follow the repo standard. Could you keep this the same with the rest of the repo?